python/composio/tools/toolset.py

```diff
@@ -977,6 +978,47 @@ def get_entity(self, id: t.Optional[str] = None) -> Entity:
         """Get entity object for given ID."""
         return self.client.get_entity(id=id or self.entity_id)
 
+    def get_expected_params(
```

Comment:
The error message in `get_expected_params` method might be more informative if it included the expected auth schemes available for the app, rather than just stating the requested scheme is not found. This would aid developers in debugging.